pages/video/[id].tsx

```diff
@@ -0,0 +1,49 @@
+import { Text, Flex } from "@chakra-ui/react";
+import { useRouter } from "next/router";
+import React from "react";
+import VideoPlayer from "../../components/login/videoPlay";
+import axios from "axios";
+
+type Params = {
+  params: {
+    id: string;
+  };
+};
+
+export async function getStaticProps({ params }: Params) {
```

Comment:
Because the only thing you are statically returning in `getStaticProps` is the `id` (which you can get from `useRouter()`), I would remove `getStaticProps` as well as `getStaticPaths` in favour of:
```
const router = useRouter()
const { id } = router.query
```
Won't need `isFallback` either.

Reply:
Yes, was actually struggling to make a decision here, but decided to go with the above code since I thought **Incremental Static Generation** would be more promising.

This [doc](https://static-tweet.vercel.app/) says that, if a page has never been rendered, it will display a skeleton.

Would mean a lot to me, if you take a look at the doc and help me make the conclusion.